src/main/java/edu/kis/powp/command/FiguresJoeFactoryCommand.java

```diff
@@ -0,0 +1,48 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.util.ArrayList;
+
+public class FiguresJoeFactoryCommand implements Job2dDriver {
```

Comment:
Actually, it would be very easy to convert this solution to a single generic functionality (to cover building command from any script). Actually it is even possible now, because we can externally execute any set of commands on FiguresJoeFactoryCommand  (but the name doesn't suggest such possibility).